recipe/meta.yaml

```diff
@@ -11,40 +11,59 @@ source:
   sha256: b976c7f962e907c34cfa261e4346b28bcbee69db12880c249745f537a5b4be0b
 
 build:
-  number: 0
+  number: 1
   noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
-
-requirements:
-  host:
-    - pip
-    - python >=3.6
-  run:
-    - colour
-    - python-graphviz >=0.9
-    - lightgbm
-    - matplotlib-base
-    - numpy
-    - pandas
-    - pillow
-    - pyspark
-    - pytest
-    - python >=3.6
-    - scikit-learn
-    - py-xgboost
-
-test:
-  imports:
-    - dtreeviz
-    - dtreeviz.models
-  commands:
-    - pip check
-  requires:
-    - pip
+  script: python -m pip install . -vv
+
+
+outputs:
+  - name: {{ name }}
+    requirements:
+      host:
+        - pip
+        - python >=3.6
+      run:
+        - python >=3.6
+        - colour
+        - python-graphviz >=0.9
+        - matplotlib-base
+        - numpy
+        - pandas
+        - pillow
+        - pytest
+        - scikit-learn
+   
+    test:
+      imports:
+        - dtreeviz
+      commands:
+        - pip check
+      requires:
+        - pip
+  - name: {{ name }}-extended
+    requirements:
+      host:
+        - pip
+        - python >=3.6
+      run:
+        - python >=3.6
+        - {{ pin_subpackage(name, max_pin='x.x.x.x.x.x') }}
```

Comment:
Not familiar with this but just looked it up.
https://docs.conda.io/projects/conda-build/en/latest/resources/variants.html#referencing-subpackages
I'm fine if you are happy with using using the max_pin arg

Reply:
@raybellwaves I've just used the same convention as in airflow recipe from your [comment](https://github.com/parrt/dtreeviz/issues/144#issuecomment-859656476). Now, I've tried with flag `exact=True` I think it should work as well (there is only one version of `dtreeviz` in the build).